bin/slack-notify.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+source ${SLACK_NOTIFIER_TEMPLATE}
+
+echo "${ACTION}"
```

Comment:
If you want to add some output like this, that's cool, but add some detail like "Action: ${ACTION}`